docs/create-packages/Symbol-Packages-snupkg.md

```diff
@@ -16,15 +16,15 @@ ms.reviewer:
 
 # Creating symbol packages (.snupkg)
 
-Symbol packages allow you to improve the debugging experience of your NuGet packages.
+A good debugging experience relies on the presence of debug symbols as they provide critical information like the association between the compiled and the source code, names of local variables, stack traces, and more. By default, these symbols are are built as separate files to minimize the size of shipping binaries. You can use symbol packages (.snupkg) to distribute these symbols and improve the debugging experience of your NuGet packages.
```

Comment:
should we also call - to minimize the size of the nupkg?

Reply:
Yeah that’s probably more important than explaining non-embedded PDBs. I change it up.

Reply:
I ended up removing the comment as it felt like a little too low level a detail for an introduction of symbol packages.